_members/04-kaylyn-clark.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Kaylyn Clark
+image: images/kaylyn_photo.jpg
+description: Postdoctoral Fellow
+role: postdoc
+aliases: 
+links:
+  github: [https://github.com/KayCla](url)
+  orcid: [0000-0002-7207-6608](https://orcid.org/0000-0002-7207-6608)
+  LinkedIn: [https://www.linkedin.com/in/kaylyn-clark-phd-766300101/](url) 
```

Comment:
```suggestion
  linkedin: kaylyn-clark-phd-766300101 
```

Sorry this was not intuitive - these come from [here](https://github.com/rokitalab/rokitalab.com/blob/103bcb537a567941aad47df49281e9263cff8bd4/_data/types.yaml#L127-L131), are case-sensitive, and take values only